akka-actor/src/main/scala/akka/io/Dns.scala

```diff
@@ -66,12 +92,21 @@ object Dns extends ExtensionId[DnsExt] with ExtensionIdProvider {
         addresses.iterator.collect { case a: Inet6Address => a }.to(immutable.IndexedSeq)
       Resolved(name, ipv4, ipv6)
     }
+
+    @deprecated("cached(DnsProtocol.Resolve)", "2.6.0")
```

Comment:
Capitalize all of these
```suggestion
    @deprecated("Cached(DnsProtocol.Resolve)", "2.6.0")
```

Reply:
how come? it is a method reference, maybe prefix with Use?